playwright/_impl/_api_structures.py

```diff
@@ -119,3 +120,23 @@ class FilePayload(TypedDict):
     name: str
     mimeType: str
     buffer: bytes
+
+
+__all__ = [
```

Comment:
Why do we need this here?

Reply:
Since here we are re-exporting the devices and if there isn't __all__ then flake8 would complain